smk/QC_2.smk

```diff
@@ -361,6 +369,14 @@ rule qc2_host_filter:
         ) >& {log}
         """
 
+# derived rule: metaT - output is temporary
+use rule qc2_host_filter as qc2_host_filter_metaT with:
+    output:
+        host_free_fw=temp("{wd}/{omics}/4-hostfree/{sample}/{run}.1.fq.gz"),
```

Comment:
@jszarvas If we upload to SRA or the like, is it better to leave the rRNA reads or remove them? If we hardcode it as `temp()`, then the users will not have access to host-free reads that also still retain rRNA sequences.